setup.py

```diff
@@ -76,9 +76,14 @@ def platform_suffix(self):
     def mavsdk_server_filepath(self):
         """
         The location of the downloaded `mavsdk_server` binary
-        """
+        For windows this needs to be a .exe file
+        """   
+        if sys.platform.startswith('win'):
+            return 'mavsdk/bin/mavsdk_server.exe'
         return 'mavsdk/bin/mavsdk_server'
```

Comment:
Nitpick: I think this is slightly more readable:

```suggestion
        if sys.platform.startswith('win'):
            return 'mavsdk/bin/mavsdk_server.exe'
        else:
            return 'mavsdk/bin/mavsdk_server'
```

Reply:
I had It like your change before and it was suggested I change it to the other one 😆